### PR TITLE
Add php 8.3 and remove php 8.0

### PR DESCRIPTION
**Pull request by Luke-Roy-IBM**
Remove PHP:8.0 as it is deprecated
Add new PHP:8.3 runtime

### CODE REVIEW

File: core/php8.3Action/CHANGELOG.md [hunk 1]
```diff
@@ -18,23 +18,10 @@
 -->
 
 ## Apache 1.19.0
```
Comment (dgrove-oss) [inline core/php8.3Action/CHANGELOG.md:20]:
```suggestion
## Initial Release
```

File: core/php8.3Action/CHANGELOG.md [hunk 2]
```diff
@@ -18,23 +18,10 @@
...
-  - No change
 
-## Apache 1.18.0
-  - Use php:8.0-cli-buster image to always pull latest patch version
-  - Golang Action loop updatetd to golang 1.20
-  - Support array result include sequence action (#120)
+- Initial release
```
Comment (dgrove-oss) [inline core/php8.3Action/CHANGELOG.md:22]:
```suggestion
```

**Review by dgrove-oss — CHANGES_REQUESTED** [review]
small tweak to future release note; otherwise looks good

**Comment by Luke-Roy-IBM** [comment]
@dgrove-oss fixes applied 👍

**Review by dgrove-oss — APPROVED** [review]
lgtm

**Comment by akrabat** [comment]
Thanks @Luke-Roy-IBM!

**Pull request merged**